test/org/camunda/latera/bss/utils/NumericSpec.groovy

```diff
@@ -9,171 +9,200 @@ class NumericSpec extends Specification {
     Numeric.toIntSafe(input) == result
 
     where:
-    input|result
-    0    |0
-    1    |1
-    1.1  |1
-    '1'  |1
-    '1.1'|1
-    '1,1'|1
-    'a'  |null
-    null |null
-    false|null
-    []   |null
-    [:]  |null
+    input |result
+    0     |0
+    1     |1
+    1.1   |1
+    '0'   |0
+    '1'   |1
+    '01'  |1
+    '1.1' |1
+    '1,1' |1
+    '01.1'|1
+    '01,1'|1
```

Comment:
personally i would expect exception here, but okay it is SAFE, so i would expect ```null```

anyway it's up to you

Reply:
toIntSafe takes 2 arguments - first is value should be converted to BigInteger, and second is default value which is returned if it's not possible.
Second argument is null by default. There is separated test for non-null input.